README.md

```diff
@@ -77,24 +77,24 @@ Tests may be run with ```sbt test```.
 
 ## Command line
 
-`java -jar target/scala-2.11/dagr-0.1.0-SNAPSHOT.jar` to see the full list of options.
+`java -jar target/scala-2.11/dagr-0.1.1-SNAPSHOT.jar` to see the full list of options.
```

Comment:
I see. ok.

On Sat, Sep 10, 2016 at 2:27 PM, Nils Homer <notifications@github.com>
wrote:

> In README.md
> <https://github.com/fulcrumgenomics/dagr/pull/215#discussion_r78281685>:
>
> > @@ -77,24 +77,24 @@ Tests may be run with ```sbt test```.
> >
> >  ## Command line
> >
> > -`java -jar target/scala-2.11/dagr-0.1.0-SNAPSHOT.jar` to see the full list of options.
> > +`java -jar target/scala-2.11/dagr-0.1.1-SNAPSHOT.jar` to see the full list of options.
>
> 0.1.0 is the latest release.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fulcrumgenomics/dagr/pull/215/files/60af86f9330df674d56a4b33f436a07ec4765f15#r78281685>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACnk0lfJBNrWKOwrCIMDmJHCFgfs1eJWks5qovavgaJpZM4J5yQn>
> .
>